libcloud/loadbalancer/drivers/dimensiondata.py

```diff
@@ -559,8 +560,8 @@ def ex_create_virtual_listener(self,
                                    network_domain_id,
                                    name,
                                    ex_description,
-                                   port,
                                    pool,
+                                   port=None,
```

Comment:
But you're changing the order of variables.  If keyword args aren't used here it would break.